src/community/advanced-data-card.tsx

```diff
@@ -0,0 +1,462 @@
+        const finalActions = useTopActions(actions, onClose);
+        const hasAcations = finalActions?.length > 0;
+
+        const footerProps = {button, footerImage, footerText, footerTextLinesMax, buttonLink};
+
+        const extraSpaceSize = smallSlotSpace ? 8 : 24;
+
+        const topActionsStylesWithIcon = {position: 'absolute', top: 8, right: 8, zIndex: 2} as const;
+
+        const hasFooter = !!button || !!footerImage || !!footerText || !!buttonLink;
+
+        return (
+            <section
+                aria-label={ariaLabel}
+                className={sprinkles({
+                    position: 'relative',
+                    height: '100%',
+                })}
+            >
+                <Boxed
+                    className={styles.boxed}
+                    dataAttributes={{'component-name': 'AdvancedDataCard', ...dataAttributes}}
+                    ref={ref}
+                    width="100%"
+                    height="100%"
+                >
+                    <div className={styles.dataCard}>
+                        <Touchable
+                            onPress={onPress}
+                            tabIndex={0}
+                            maybe
+                            className={sprinkles({
+                                position: 'relative',
+                                height: '100%',
+                            })}
+                        >
+                            <div className={onPress ? styles.interaction : ''}>
+                                <div
+                                    className={classNames(
+                                        styles.cardContentStyle,
+                                        !hasFooter && !extra ? styles.minHeight : ''
+                                    )}
+                                >
+                                    <Box paddingTop={8}>
+                                        <Stack space={8} className={sprinkles({flex: 1})}>
+                                            {stackingGroup && (
+                                                <div
+                                                    className={sprinkles({
+                                                        display: 'flex',
+                                                        width: '100%',
+                                                    })}
+                                                >
+                                                    <div
+                                                        className={sprinkles({
+                                                            width: '100%',
+                                                        })}
+                                                    >
+                                                        {stackingGroup}
+                                                    </div>
+                                                </div>
+                                            )}
```

Comment:
are these divs really needed? isn't this enough?
```suggestion
                                            {stackingGroup}
```